webapp/sources/ldap-inventory/inventory-fusion/src/main/scala/com/normation/inventory/provisioning/fusion/FusionInventoryParser.scala

```diff
@@ -944,7 +944,7 @@ class FusionInventoryParser(
       d = optText(e \ "DESCRIPTION")
       sev = optText(e \ "SEVERITY" ).map(SoftwareUpdateSeverity.parse)
       date = optText(e \ "DATE")
-      ids = (e \ "ID").map(_.text).toList
+      ids = (e \ "ID").toList.flatMap(_.text.split(","))
```

Comment:
;(

Reply:
I gave up on perl/fusion

Reply:
Btw, this fix also support the old format ;) So when you'll be brave enough to fight with fusion .... :D